pootle/apps/pootle_store/models.py

```diff
@@ -2044,22 +2044,20 @@ def serialize(self):
 
         return ret
 
-    def sync(self, update_structure=False, conservative=True,
-             user=None, skip_missing=False, only_newer=True):
+    def sync(self, force=False, user=None, skip_missing=False):
         """Sync file with translations from DB."""
         if skip_missing and not self.file.exists():
             return
 
         last_revision = self.get_max_unit_revision()
 
-        #TODO only_newer -> not force
-        if (only_newer and self.file.exists() and
+        if (not force and self.file.exists() and
             self.last_sync_revision >= last_revision):
             logging.info(u"[sync] No updates for %s after [revision: %d]" %
                 (self.pootle_path, self.last_sync_revision))
             return
 
-        if not self.file.exists() and not skip_missing:
```

Comment:
@unho im glad you asked 8)

because in L2049 it tests for `skip_missing and not self.file.exists()` and returns if it exists - its superfluous